### DISCUSSION TITLE
本地验证码是不是测试通过就可以用了

### DISCUSSION THREAD

**Discussion by Hanabiiiiii** [mikumifa/biliTickerBuy#304] [提问专区]
![屏幕截图 2024-06-30 224740](https://github.com/mikumifa/biliTickerBuy/assets/122472858/c6ffdc72-d7fa-4587-b0f0-c56e0f500de3)

**Comment by mikumifa** [comment]
这是正常的

**Reply by Hanabiiiiii** [reply]
那这本地过码直接用就可以是吧
